test/src/test/java/jenkins/security/Security3430Test.java

```diff
@@ -126,7 +129,7 @@ private static void _run(JenkinsRule j, String agentName, String expectedRemotin
                 if (requestingJarFromAgent) {
                     assertThat(logRecords, hasItem(logMessageContainsString("Allowing URL: file:/")));
                 } else {
-                    assertThat(logRecords, is(empty()));
+                    assertThat(logRecords.stream().map(LogRecord::getMessage).toList(), is(empty()));
```

Comment:
Previous failure message was unhelpful (`LogRecord` does not override `toString`).